vendor/composer/installed.json

```diff
@@ -1,5 +1,76 @@
 {
```

Comment:
Drush cim throws an error during import: 

```
 [error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Unable to install the <em class="placeholder">admin_toolbar</em> module since it does not exist.
Unable to install the <em class="placeholder">admin_toolbar_tools</em> module since it does not exist.
Unable to install the <em class="placeholder">blazy</em> module since it does not exist.
Unable to install the <em class="placeholder">slick</em> module since it does not exist.
Unable to install the <em class="placeholder">bootstrap</em> theme since it does not exist. in Drupal\Core\Config\ConfigImporter->validate() (line 748 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php). 

```